CompuCell3D/core/CompuCell3D/plugins/Secretion/FieldSecretor.cpp

```diff
@@ -772,3 +772,37 @@ bool FieldSecretor::uptakeInsideCellAtCOM(CellG * _cell, float _maxUptake, float
 
 
 }
+
+float FieldSecretor::_amountSeenByCell(CellG * _cell) {
+
+    if (!pixelTrackerPlugin) {
+        // ASSERT_OR_THROW("PixelTracker Plugin has been turned off. Cannot execute _amountSeenByCell function",pixelTrackerPlugin);
+        
+        return -1.0;

```

Comment:
Can't users encounter this and become confused?

Reply:
This is handled in SWIG file. But  good point, perhaps we should also keep it here if somebody decides to use if from a steppable written in C++. For, now i think it is safe to assume it will be called via python interface though